test/specs/views/Stastistic/StatisticGroup-test.js

```diff
@@ -1,15 +1,13 @@
-import faker from 'faker'
 import React from 'react'
 import * as common from 'test/specs/commonTests'
-import numberToWord from 'src/utils/numberToWord'
-
 import StatisticGroup from 'src/views/Statistic/StatisticGroup'
 
 describe('StatisticGroup', () => {
   common.isConformant(StatisticGroup)
   common.hasUIClassName(StatisticGroup)
   common.rendersChildren(StatisticGroup)
   common.propKeyOnlyToClassName(StatisticGroup, 'horizontal')
+  common.propValueOnlyToClassName(StatisticGroup, 'widths')
```

Comment:
This test is _mostly_ accurate, but not entirely.  Consider when a user passes `widths='3'`, the value `'3'` is not actually added to the className, the word `three` is.

We should likely add a common test for `implementsNumberToWordProp(Statistic, 'widths')`.  This test would assert that the `numberToWord` value is added to the `className`.  It would also assert that it works with numbers, strings, and number words (i.e. 'two').

The reason I think we should allow the second param to this common tests is that on other components, like `FormFields`, the `numerToWord` prop is `width`.  This way, we can point this test to a specific prop name.